docs/usage/shoot_access.md

```diff
@@ -0,0 +1,75 @@
+
+The username associated with such `kubeconfig` will be the same which is used for authenticating to the Gardener API. Apart from this advantage, the created `kubeconfig` will not be persisted anywhere.
+
+In order to request such a `kubeconfig`, you can run the following commands:
+
+```bash
+export NAMESPACE=my-namespace
+export SHOOT_NAME=my-shoot
+kubectl create \
+    -f <path>/<to>/kubeconfig-request.json \
+    --raw /apis/core.gardener.cloud/v1beta1/namespaces/${NAMESPACE}/shoots/${SHOOT_NAME}/adminkubeconfig | jq -r ".status.kubeconfig" | base64 -d
+```
+
+Here, the `kubeconfig-request.json` has the following content:
+
+```json
+{
+    "apiVersion": "authentication.gardener.cloud/v1alpha1",
+    "kind": "AdminKubeconfigRequest",
+    "spec": {
+        "expirationSeconds": 1000
+    }
+}
+```
+
+> The [`gardenctl-v2`](https://github.com/gardener/gardenctl-v2/) tool makes it easy to target shoot clusters and automatically renews such `kubeconfig` when required.
+
+## OpenID Connect
+
+The `kube-apiserver` of shoot clusters can be provided with [OpenID Connect configuration](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens) via the `ShootSpec`:
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+...
+spec:
+  kubernetes:
+    oidcConfig:
+      ...
+```
+
+It is the end-user's responsibility to incorporate the OpenID Connect configurations in `kubeconfig` for accessing the cluster (i.e., Gardener will not automatically generate `kubeconfig` based on these OIDC settings).
+The recommended way is using the `kubectl` plugin called [`kubectl oidc-login`](https://github.com/int128/kubelogin) for OIDC authentication.
+
+If you want to use the same OIDC configuration for all your shoots by default then you can use the `ClusterOpenIDConnectPreset` and `OpenIDConnectPreset` API resources. They allow defaulting the `.spec.kubernetes.kubeAPIServer.oidcConfig` fields for newly created `Shoot`s such that you don't have to repeat yourself every time (similar to `PodPreset` resources in Kubernetes).
+`ClusterOpenIDConnectPreset` specified OIDC configuration applies to `Projects` and `Shoots` cluster-wide (hence, only available to Gardener operators) while `OpenIDConnectPreset` is `Project`-scoped.
+Shoots have to "opt-in" for such defaulting by using the `oidc=enable` label.
+
+For further information on `(Cluster)OpenIDConnectPreset`, refer to [this document](https://github.com/gardener/gardener/blob/master/docs/usage/openidconnect-presets.md).
```

Comment:
I guess the relative link is the preferred way. Similar changes in other places.
```suggestion
For further information on `(Cluster)OpenIDConnectPreset`, refer to [this document](openidconnect-presets.md).
```

Reply:
/cc @rfranzke